### PR TITLE
Update to 0.8.8

### PR DESCRIPTION
**Pull request by paulbellamy**
Also brought in commits from upstream

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,14 +13,10 @@ CMD ["--help"]
 ADD prom-run /
 
 RUN apk update && \
-    apk add bash \
-    ca-certificates \
-    git \
```
Comment (tomwilkie) [inline Dockerfile]: Removing this will break terradiff.

Comment (paulbellamy) [inline Dockerfile]: Removing what?

Comment (tomwilkie) [inline Dockerfile]: I thought you'd removed git.  They were put one per line to make it clearer what was there and what wasn't.

**Comment by paulbellamy** [comment]
Yeah, the merge from upstream commits, moved them back all onto one line.
happy to do it either way.

**Comment by tomwilkie** [comment]
I'd abandon upstream at this point, there is enough customisation in here.  Please move back to one line.

**Comment by paulbellamy** [comment]
done

**Comment by tomwilkie** [comment]
LGTM

**Pull request merged**